docs/how-tos/provision-setup.md

```diff
@@ -314,15 +314,15 @@ sudo ./preinstall.sh /path/to/rootfs
 {{< tablestep >}}
 **1. Create and download files**
 
-Run the following commands to create directories for the provisioning binaries, then download the binaries and make them executable:
+As root on the target device, run the following commands to create directories for the provisioning binaries, then download the binaries and make them executable:
```

Comment:
So the issue is that these instructions are supposed to be when you're NOT on the target device. This is pre-install to a factory image that's not currently running. On a live device, the normal install script should just be used.

Reply:
oh that makes my experience make more sense. this is the manual version of the **preinstall** flow

I'm guessing these commands are wrong if you're not on the target device bc of abs paths

Reply:
So do any of these comments apply?

Reply:
nope I'm totally wrong

but I think these would still fail in 'editing an sd card' context because of the mix of abs and relative paths

see discussion below about removing the 'manual install' tab entirely -- @npentrel are you okay with that change?